average.js

```diff
@@ -1,6 +1,7 @@
 
 function average(numbers) {
-  return numbers.reduce((p, c)=> p + c, 0) / numbers.length;
+  const newArray = numbers.filter( value => !Number.isNaN(value) );
```

Comment:
Nice translation of functionality to concise code +1
It's important to name our variables in a way that readers know their intent without reading all of the code.
Reflect on the intention of `newArray` and try giving it a more descriptive name.